python/lsst/pipe/tasks/transformMeasurement.py

```diff
@@ -119,32 +119,18 @@ def run(self, inputCat, wcs, calib):
         return outputCat
 
 
-class RunTransformConfig(pexConfig.Config):
-    """!Configuration for RunTransformTask."""
+class RunTransformConfigBase(object):
+    """!Configuration for RunTransformTaskBase derivatives."""
     transform = pexConfig.ConfigurableField(
         doc="Subtask which performs transformations",
         target=TransformTask
     )
-    measConfig = pexConfig.Field(
-        dtype=str,
-        doc="Dataset type of measurement operation configuration",
-        default="processCcd_config"
-    )
-    measType = pexConfig.ChoiceField(
-        dtype=str,
-        doc="Type of measurement operation performed",
-        default="SingleFrame",
-        allowed={
-            "SingleFrame": "Single frame measurement",
-            "Forced": "Forced measurement"
-        }
-    )
 
 
-class RunTransformTask(pipeBase.CmdLineTask):
+class RunTransformTaskBase(object):
```

Comment:
I do think single inheritance is cleaner here, and it's a pattern we already use regularly (see `ProcessImageTask`).  Declaring that the class is abstract and should not be used directly in the documentation (along with having "Base" in the name somewhere) should be sufficient to avoid confusion.